src/get-app-authentication.ts

```diff
@@ -1,12 +1,17 @@
 import { githubAppJwt } from "universal-github-app-jwt";
 
-import { AppAuthentication } from "./types";
+import { AppAuthentication, State } from "./types";
 
-export async function getAppAuthentication(
-  id: number,
-  privateKey: string
-): Promise<AppAuthentication> {
-  const appAuthentication = await githubAppJwt({ id, privateKey });
+export async function getAppAuthentication({
+  id,
+  privateKey,
+  timeDifference,
+}: State): Promise<AppAuthentication> {
+  const appAuthentication = await githubAppJwt({
+    id,
+    privateKey,
+    now: timeDifference,
```

Comment:
Are you still working on this?

I'm a bit confused that we are setting `now` which is a UNIX epoch as milliseconds to `timeDifference` which seems to be a time difference in seconds? Or am I missing something?

Reply:
Ah sorry I missed that, the `githubAppJwt` value for `now` should be the current time in seconds, not the difference in seconds from the GitHub time. I'm adding a fix for that right now.